src/api/scopes/controllers/get-scopes-for-user.js

```diff
@@ -10,7 +10,10 @@ function isUserInAServiceTeam(teamIds, userGroups) {
 const getScopesForUserController = {
   options: {
     auth: {
-      strategy: 'azure-oidc'
+      strategy: 'azure-oidc',
+      access: {
+        scope: [config.get('oidcAdminGroupId')]
+      }
```

Comment:
Sorry, is this right? only admin? I think all users need to be able to get their scopes?